### PR TITLE
Longw/arc automation pipeline ev2

### PR DESCRIPTION
**Pull request by wanlonghenry**
arc automation pipeline ev2

### CODE REVIEW

File: deployment/arc-k8s-extension-release-v2/ServiceGroupRoot/Scripts/arcExtensionReleaseCanaryPreview.sh [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+if [ -z "$IS_CUSTOMER_HIDDEN" ]; then
+    echo "-e error is_customer_hidden must be provided "
+    exit 1
+fi
+if [ -z "$CHART_VERSION" ]; then
+    echo "-e error chart version must be provided "
+    exit 1
+fi
+
+MCR_NAME_PATH="mcr.microsoft.com/azuremonitor/containerinsights/canary/preview/azuremonitor-containers"
```
Comment (ganga1980) [inline deployment/arc-k8s-extension-release-v2/ServiceGroupRoot/Scripts/arcExtensionReleaseCanaryPreview.sh:32]: i see chart verified only against canary preview repo path, this should be checked against release stage, for example if its canary check against canary repo path and if its stable release stage, then check again release stage path, right?

Comment (wanlonghenry) [reply deployment/arc-k8s-extension-release-v2/ServiceGroupRoot/Scripts/arcExtensionReleaseCanaryPreview.sh]: updated

**Comment by wanlonghenry** [comment]
> Can you confirm that these changes will allows the following scenarios:
> 
> * Release specific version to canary
> * Release specific version to prod regions
> * Release specific version to specific region
> * Release specific version to specific release train (stable, preview or any other test train)
> * Release any combination of above cases

This is mainly for release pipeline, for customized target release, I would suggest not mixing them since for arc api put request, one condition is that each new put, all the previous regions must be included.

**Comment by wanlonghenry** [comment]
new release pipelines set up: https://github-private.visualstudio.com/microsoft/_release?definitionId=67&view=mine&_a=releases
https://github-private.visualstudio.com/microsoft/_release?definitionId=68&view=mine&_a=releases
https://github-private.visualstudio.com/microsoft/_release?definitionId=69&view=mine&_a=releases

**Review by ganga1980 — DISMISSED** [review]

**Review by ganga1980 — APPROVED** [review]

**Review by jatakiajanvi12 — APPROVED** [review]

**Pull request merged**